client/src/components/core/ms-filter/types.ts

```diff
@@ -0,0 +1,13 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+export interface MsFilterLabels {
```

Comment:
I would mark them as optional (`statusActive?: boolean`) and consider undefined as false, which makes it easier to call the function
```typescript
getVisibleUsers({statusActive: true, profileStandard: true});
```
instead of having to list them all.